app/helpers/application_helper.rb

```diff
@@ -4,7 +4,7 @@
 module ApplicationHelper
 
   def base_container
-    if controller_name == "movies"
+    if controller_name == "movies" || "monetizes"
```

Comment:
こちらは使用しなくなりましたので，戻しておきましょう。

```suggestion
    if controller_name == "movies"
```